go/kbfs/libpages/server_test.go

```diff
@@ -80,6 +109,12 @@ func TestServerDefault(t *testing.T) {
 	server.ServeHTTP(w, httptest.NewRequest("GET", "/non-existent", nil))
 	require.Equal(t, http.StatusNotFound, w.Code)
 
+	// Regression test HOTPOT-2207. Need to run with
```

Comment:
When you say "Need to run", you mean that before this fix, that env var was needed to cause the failure, right?  The way you have the comment worded now, it seems like the env var is needed to make the test pass.

Maybe just get rid of the second sentence and actually set the environment variable using `os.Setenv`?  (And unset it in a defer.)